CMakeLists.txt

```diff
@@ -75,7 +75,7 @@ if(OTIO_PYTHON_INSTALL)
         message(STATUS "OTIO Defaulting Python install to ${OTIO_RESOLVED_PYTHON_INSTALL_DIR}")
     else()
         # either python_install or install_prefix have been set
-        if(OTIO_PYTHON_INSTALL_DIR_INITIALIZED_TO_DEFAULT)
+        if(NOT OTIO_PYTHON_INSTALL_DIR)
```

Comment:
`OTIO_PYTHON_INSTALL_DIR_INITIALIZED_TO_DEFAULT` seemed to always evaluate to false and I couldn't find anything in the CMake docs referencing `_INITIALIZED_TO_DEFAULT` as a suffix. But let me know if I missed something.

Reply:
OTIO_PYTHON_INSTALL_DIR_INITIALIZED_TO_DEFAULT appears to be vestigial from an earlier iteration. 

That said, the previous clause checks for OTIO_PYTHON_INSTALL_DIR is "", and the docs say that is supposed to mean the install dir was not supplied from the command line. So how could we actually end up in this clause with it set to NO?

Reply:
We can endup here if `OTIO_PYTHON_INSTALL_DIR` is not set and `CMAKE_INSTALL_PREFIX` was not provided by the user.

So `cmake -DOTIO_PYTHON_INSTALL=ON` will get us here.

Reply:
Wait no. You can get here if you set `CMAKE_INSTALL_PREFIX`. Sorry for the confusion.

So `cmake -DCMAKE_INSTALL_PREFIX=/tmp/otio -DOTIO_PYTHON_INSTALL` for example.